libyul/backends/evm/ControlFlowGraphBuilder.cpp

```diff
@@ -76,6 +87,16 @@ void cleanUnreachable(CFG& _cfg)
 		cxx20::erase_if(node->entries, [&](CFG::BasicBlock* entry) -> bool {
 			return !reachabilityCheck.visited.count(entry);
 		});
+
+	// Remove functions which are never referenced.
```

Comment:
This change in ``cleanUnreachable`` could be split out into a separate PR and merged directly into ``develop`` up front, right?